sdk/core/System.Net.ClientModel/tests/client/Internal/ModelReaderWriterExtensions.cs

```diff
@@ -0,0 +1,272 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Net.ClientModel.Core;
+using System.Text.Json;
+
+namespace System.Net.ClientModel.Tests.Client
+{
+    internal static class ModelReaderWriterExtensions
```

Comment:
Why do we need these?  Let's have a goal to remove the full Internal folder prior to GA, per your generator strategy.